db-client-java/src/main/java/com/eventstore/dbclient/EventData.java

```diff
@@ -49,8 +49,8 @@ public static EventDataBuilder builderAsBinary(String eventType, byte[] eventDat
         return builderAsBinary(null, eventType, eventData);
     }
 
-    public static EventDataBuilder builderAsBinary(UUID id, String eventType, byte[] eventData) {
-        return EventDataBuilder.binary(eventType, eventData);
+    public static EventDataBuilder builderAsBinary(UUID eventId, String eventType, byte[] eventData) {
```

Comment:
This change is not needed.

Reply:
Do you mean `id` param is intentionally not passed to builder or renaming id->evetId is not need?

I case of rename, I just made it consistent with #builderAsJson method